src/lib/Fullscreen.js

```diff
@@ -73,33 +67,21 @@ class Fullscreen extends EventEmitter {
      * @return {boolean} In fullscreen or not
      */
     isFullscreen(element) {
-        let fullscreen;
         if (this.isSupported()) {
-            fullscreen = !!(
-                document.fullscreenElement ||
-                document.mozFullScreenElement ||
-                document.webkitFullscreenElement ||
-                document.msFullscreenElement
-            );
-        } else {
-            fullscreen = element instanceof HTMLElement && element.classList.contains(CLASS_FULLSCREEN);
+            return !!fscreen.fullscreenElement;
```

Comment:
Could we simplify this more?
From the fscreen docs
```null if not in fullscreen mode, or the DOM element that's in fullscreen mode
// replacement for: document.fullscreenElement
// mapped to: document.vendorMappedFullsceenElement
// note that fscreen.fullscreenElement uses a getter to retrieve the element
// each time the property is accessed.```

We know that this will either be null (not supported or not in fullscreen), or the element that's in fullscreen.

Reply:
Yeah, maybe swap it up and check if the fullscreen class is present, else return !!fscreen.fullscreenElement

Reply:
I'm not sure I understand these suggestions. This method is intended to check if the page is currently in fullscreen mode. If native fullscreen is supported, it coerces the `fscreen.fullscreenElement` property to return a boolean. If not, it does the same using the passed-in element's classlist property.